src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpTokenKind.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.HttpRequest;
+
+internal enum HttpTokenKind
+{
+    /*
+                              GET https://example.com/{something}
+    Word                      --- -----   ------- ---  ---------
+    Whitespace                   -
+    Punctuation                        ---       -   --         -
+    NewLine                                                      _
+    */
+    Word,
+    Whitespace,
+    Punctuation,
+    NewLine,
+    Missing
```

Comment:
Do we need this? Iirc we said we could infer missing syntax based on span (anything with 0 span is missing). Iirc, we also don't synthesize `SyntaxToken`s for missing syntax - we just create `SyntaxNode`s with 0 span for this - so I think we can remove this `TokenKind`...